Student/StudentUITests/StudentUITests.swift

```diff
@@ -16,10 +16,18 @@
 
 import XCTest
 import TestsFoundation
+import SwiftUITest
 
 class StudentUITestsEG2: StudentTest {
     func testExample() {
       EarlGrey.selectElement(with: grey_keyWindow())
         .perform(grey_tap())
     }
+
+    func testExample2() {
+        let driver = DriverFactory.getXCUITestDriver(XCUIApplication())
+        let label = "Find my school"
+        let ele = driver.find(label: label)
```

Comment:
I believe this test will fail if the user has logged into the simulator before, since it will jump straight to the dashboard.

Reply:
I updated this to be a bit more universal.

I thought/hoped the framework was resetting app state before each test run?